shopgate/pwa
# Fixed scanner route redirects

# Description
This ticket is about to fix an issue with the scanner route redirects, where query parameters got lost during the redirect process.

## Type of change


## How to test it

Open a link to the scanner route with query parameters. And verify, that the query doesn't get lost.
